.travis.yml

```diff
@@ -0,0 +1,52 @@
+if: (branch = develop) OR (branch = master) OR (type = pull_request) OR (tag IS present)
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
+  - yarn test
+  - yarn build
+after_success:
+  # Pull Request - Deploy it to a review environment
+  # Travis doesn't do deploy step with pull requests builds
+  - ./travis/deploy_pull_request.sh
+deploy:
```

Comment:
So many deployments for such a simple frontend.

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2FBs0B9jBFnU0b6%2Fgiphy.gif&token=toystory" data-gitmeme-token="toystory"><img src="https://media0.giphy.com/media/Bs0B9jBFnU0b6/giphy.gif" title="Created by gitme.me with /toystory"/></a>

Reply:
This is a cool functionality called PRaul :) 

It deploys a PR automatically, it sends a testable URL for every PR. You'll love it